saw-central/src/SAWCentral/Crucible/Common/Pred.hs

```diff
@@ -0,0 +1,27 @@
+{-# LANGUAGE DataKinds #-}
+{-# LANGUAGE FlexibleContexts #-}
+{-# LANGUAGE GADTs #-}
+{-# LANGUAGE MultiParamTypeClasses #-}
+{-# LANGUAGE OverloadedStrings #-}
+{-# LANGUAGE RankNTypes #-}
+{-# LANGUAGE ScopedTypeVariables #-}
+{-# LANGUAGE TypeFamilies #-}
+{-# LANGUAGE UndecidableInstances #-}
+{-# OPTIONS_GHC -Wno-orphans #-}
+
+module SAWCentral.Crucible.Common.Pred (
+    checkBooleanType
+    ) where
+
+import Cryptol.TypeCheck.Type (tIsBit)
+import CryptolSAWCore.TypedTerm (mkTypedTerm, ttType, ttIsMono)
+import SAWCore.SharedTerm (SharedContext, Term)
+
+-- | Ensure that the term has Cryptol type `Bit`
```

Comment:
Haddock nit pick: surrounding something with backticks in Haddock will try to hyperlink it to a Haskell identifier, but in this case, `Bit` isn't the name of a Haskell type. Surround it with `@...@` instead so that it renders it with a monospace font without hyperlinking it:

```suggestion
-- | Ensure that the term has Cryptol type @Bit@.
```